src/main/java/Entities/Restaurant.java

```diff
@@ -1,4 +1,41 @@
 package Entities;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+
 public class Restaurant {
+    private String resCategory; //Category of the restaurant
+    private String location; //Location of the restaurant
+    private Double rating; //Average rating of the restaurant from its corresponding dish ratings
+    private ArrayList<Dish> dishRatings = new ArrayList<Dish>();
```

Comment:
This instance variable could be redundant sine you can already access dish ratings through the menu and dish class